app/ssttparrangement/ArrangementController.scala

```diff
@@ -355,7 +355,14 @@ class ArrangementController @Inject() (
                          balancingPaymentDate      = lastInstalment.paymentDate,
                          totalLiability            = totalLiability.toString())
 
-    PaymentPlanRequest("SSTTP", ZonedDateTime.now.format(ISO_INSTANT), knownFact, ddInstruction, pp, printFlag = true)
+    PaymentPlanRequest(
+      "SSTTP",
+      ZonedDateTime.now.truncatedTo(java.time.temporal.ChronoUnit.MILLIS).format(ISO_INSTANT),
```

Comment:
is there somewhere in the unit tests where this field can be extracted and asserted against? I'm kind of hoping that there's some static time being used somewhere to make this easier

Reply:
Will look

Reply:
No existing unit test relevant. Have added a test. Testing regex on the timestamp